cmd/root/user_agent_command_exec_id.go

```diff
@@ -0,0 +1,16 @@
+package root
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/useragent"
+	"github.com/google/uuid"
+)
+
+func withCommandExecIdInUserAgent(ctx context.Context) context.Context {
+	// A UUID that'll will allow use to correlate multiple API requests made by
+	// the same command invocation.
+	// When we add telemetry to the CLI, this exec ID will allow allow us to
+	// correlate logs in HTTP access logs with logs in Frontend logs.
```

Comment:
It's not relevant yet; it will go stale when we do this. Better remove.

Reply:
removed